src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs

```diff
@@ -451,6 +454,19 @@ public void handle_package_result(PackageResult packageResult, ChocolateyConfigu
             }
         }
 
+        private void create_ignore_files_for_executables(string installLocation, bool is64Bit)
+        {
+            // If we are using a 64 bit architecure, we want to ignore exe's targetting x86
+            // This is done by adding a .ignore file into the package folder for each exe to ignore
+            var searchLocation = is64Bit ? "tools\\x86" : "tools\\x64";
+            var exeFiles = _fileSystem.get_files(_fileSystem.combine_paths(installLocation, searchLocation), pattern: "*.exe", option: SearchOption.AllDirectories);
+
+            foreach (var exeFile in exeFiles)
+            {
+                _fileSystem.create_file(exeFile + ".ignore");
+            }

```

Comment:
If there are only 32bit files, what is the expectation here? I think it should probably go ahead and use those files.

Reply:
@ferventcoder good catch.  I have updated this PR with a slight modification.

Reply:
Somehow your whitespace changes also snuck in.